docs/start/getting-started/fragments/next/hosting.md

```diff
@@ -1,49 +1,28 @@
-You've successfully built an Amplify app with Next.js! Now that you've built something, it's time to deploy it to the web!
+You've successfully built an Amplify app with Next.js! Now that you've built something, it's time to deploy it to the web with Amplify Console!
 
 > **Note**: Amplify Console is adding support for Server-Side Rendering (SSR).
 > [Vote & comment on this issue](https://github.com/aws-amplify/amplify-console/issues/412) to show your support!
 
-## Using [Serverless Next.js Component](https://github.com/serverless-nextjs/serverless-next.js)
+## Add hosting to your app
+Next.js SSR apps can only be set up with automatic continuous deployments. In this guide we'll cover how to deploy and host your SSR web apps to quickly share with others. If you want to learn more about continuous deployment, please see [this guide](https://docs.aws.amazon.com/amplify/latest/userguide/multi-environments.html#standard).
```

Comment:
You can deploy Next.js SSR apps with AWS Amplify's CI/CD and hosting service.

Reply:
I added this language in my updated commit.